### DISCUSSION TITLE
useWatch "disable" option removed?

### DISCUSSION THREAD

**Discussion by ahmedrafayat** [react-hook-form/react-hook-form#6791] [Q&A] [answered]
According to the latest docs, useWatch still has a disabled option https://react-hook-form.com/api/usewatch

I am currently using version 7.12.2 with typescript, and it says this option does not exist. I also tried setting disable to true, but it still seems to be watching. Not sure whats wrong

**Comment by bluebill1049** [comment, marked as answer]
can you share a codesandbox? option is still there
![Screen Shot 2021-10-13 at 7 25 36 pm](https://user-images.githubusercontent.com/10513364/137096090-ff8a7c24-e791-4bb1-bd70-331b66aa29ac.png)

**Comment by ahmedrafayat** [comment]
My bad, I was using an older version. Typed in 7.12.2 instead of 7.17.2 😓 , noob mistake. But I think the doc needs an update. Should be "disabled" instead of "disable".

**Reply by bluebill1049** [reply]
all good, send us a PR if you have some spare time 👍  good pick up.
